2018.csv

```diff
@@ -21,14 +21,14 @@ DjangoCon Europe,2018-05-23,2018-05-27,"Heidelberg, Germany",2018-03-01,2018-03-
 PyCon CZ,2018-06-01,2018-06-03,"Prague, Czech Republic",2018-02-28,2018-02-28,https://cz.pycon.org/2018,https://cz.pycon.org/2018/proposals,https://cz.pycon.org/2018/sponsors
 PyLondinium,2018-06-08,2018-06-10,"London, UK",,2018-04-15,https://pylondinium.org/,https://www.papercall.io/pylondinium,https://pylondinium.org/#sponsors
 SciPy,2018-07-09,2018-07-15,"Austin, Texas, USA",2018-02-09,2018-02-09,https://scipy2018.scipy.org,https://scipy2018.scipy.org/ehome/299527/648140/,https://scipy2018.scipy.org/ehome/299527/648129/
-EuroPython,2018-07-23,2018-07-29,"Edinburgh, UK",,,https://ep2018.europython.eu/,,https://ep2018.europython.eu/sponsor
-PyOhio,2018-07-28,2018-07-29,"Columbus, Ohio, USA",,,https://pyohio.org,,
+EuroPython,2018-07-23,2018-07-29,"Edinburgh, UK",,2018-05-20,https://ep2018.europython.eu/,,https://ep2018.europython.eu/sponsor
+PyOhio,2018-07-28,2018-07-29,"Columbus, Ohio, USA",,2018-05-15,https://pyohio.org,,
 Flask Conf Brasil,2018-08-24,2018-08-25,"São Paulo, BR",2018-08-24,2018-06-30,http://flask.python.org.br/,http://flask.python.org.br/,http://flask.python.org.br/
-PyCon Australia,2018-08-24,2018-08-28,"Sydney, NSW, Australia",,,http://2018.pycon-au.org,,
+PyCon Australia,2018-08-24,2018-08-28,"Sydney, NSW, Australia",,2018-04-23,http://2018.pycon-au.org,,
 EuroSciPy,2018-08-28,2018-09-01,"Trento, Italy",2018-05-13,2018-05-13,https://www.euroscipy.org/2018/,https://pretalx.com/euroscipy18/,
 PyCon UK,2018-09-15,2018-09-19,"Cardiff, UK",,,,,
 PyGotham,2018-10-05,2018-10-06,"New York, NY, USA",,2018-05-20,https://2018.pygotham.org,https://www.papercall.io/pygotham-2018,https://2018.pygotham.org/sponsors/prospectus/
-DjangoCon US,2018-10-14,2018-10-19,"San Diego, CA, USA",,,https://2018.djangocon.us,,
+DjangoCon US,2018-10-14,2018-10-19,"San Diego, CA, USA",,2018-06-04,https://2018.djangocon.us,,
```

Comment:
[DjangoCon's website](https://2018.djangocon.us/speaking/) says 2018-06-03 anywhere on earth. Is this meant to signify the following midnight?

Reply:
I'm not sure if this is meant to be in a particular time zone or what. I put the 4th because their papercall site told me the 4th, though obviously that means "in my time zone".  I guess everyone else lists the last day, so I'll change this to the 3rd.